### PR TITLE
Added logger.exception in sspl_ll_d file for handling error traceback.

### PR DESCRIPTION
**Pull request by shriya-deshmukh**
## Problem Statement
<pre>
  <code>
    Story Ref (if any):
    Added logger.exception in sspl_ll_d file for handling error traceback.
  </code>
</pre>
## Problem Description
<pre>
  <code>
    Please add Problem decription here...
  </code>
</pre>
## Solution
<pre>
  <code>
    Please add summary of the change,List any dependencies that are required for this change.
  </code>
</pre>
## Sanity testing on RPM done
<pre>
  <code>
    - [ ] Yes
  </code>
</pre>
## Unit/Manual Testing Description
<pre>
  <code>
    Please describe the tests that you ran to verify your changes.
  </code>
</pre>

### CODE REVIEW

**Review by mariyappanp — APPROVED** [review]

File: low-level/framework/sspl_ll_d [hunk 1]
```diff
@@ -819,7 +819,7 @@ if __name__ == "__main__":
         main(conf_reader, systemd_support)
 
     except (Exception) as err:
-        logger.debug(f"{traceback.print_exc()}")
+        logger.exception(f"{traceback.print_exc()}")
```
Comment (sandeepsgt) [inline low-level/framework/sspl_ll_d:822]: When you use logger.exception, exception info automatically gets added.
So you can directly do logger.exception("Error while initializing")

Comment (shriya-deshmukh) [reply low-level/framework/sspl_ll_d]: Done.

**Review by cdeshmukh — APPROVED** [review]

File: low-level/framework/sspl_ll_d [hunk 2]
```diff
@@ -819,7 +819,7 @@ if __name__ == "__main__":
         main(conf_reader, systemd_support)
 
     except (Exception) as err:
-        logger.debug(f"{traceback.print_exc()}")
+        logger.exception("Error while initiliazing sspl-ll service : ")
         logger.critical("While spawning sspl-ll process :%r" % err)
```
Comment (thavanathan) [inline low-level/framework/sspl_ll_d:823]: We can remove the critical logging as we are already logging it with logger.exeption.

Comment (shriya-deshmukh) [reply low-level/framework/sspl_ll_d]: Done.

**Review by cdeshmukh — APPROVED** [review]

**Comment by shriya-deshmukh** [comment]
@cla-bot

**Pull request merged**